.github/workflows/validate_pyinstaller.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v4
         with:
-          python-version: "3.11"
+          python-version: "3.7"
```

Comment:
I think we can bump this to 3.8 right?

Reply:
We can, not sure it matters too much here. I'll raise a quick follow-up PR to update that.